src/Utility.cpp

```diff
@@ -357,24 +357,30 @@ String format(const char *input, ...)
     return ret;
 }
 
+unsigned long lastLog = 0;
+
 void logv(int levelFlags, String input, ...)
 {
     if ((levelFlags & DEBUG_LEVEL) != 0)
     {
-        unsigned long now = millis();
+        unsigned long now   = millis();
+        unsigned long delta = now - lastLog;
         va_list argp;
         va_start(argp, input);
     #if BUFFER_LOGS == true
         addToLogBuffer(formatArg(input.c_str(), argp));
     #else
         DEBUG_SERIAL_PORT.print("[");
         DEBUG_SERIAL_PORT.print(String(now));
-        DEBUG_SERIAL_PORT.print("]:");
+        DEBUG_SERIAL_PORT.print("] ms:{");
+        DEBUG_SERIAL_PORT.print(String(delta));
+        DEBUG_SERIAL_PORT.print("} ms:");
         DEBUG_SERIAL_PORT.print(String(freeMemory()));
-        DEBUG_SERIAL_PORT.print(": ");
+        DEBUG_SERIAL_PORT.print(" B: ");
```

Comment:
Why is there a `B` here?

Reply:
Indicates 'Bytes'

Reply:
Gotcha, I'd delete the space before then. `1024B` makes a bit more sense than `1024 B`